### PR TITLE
[Enhancement] Create dummy storage descriptor for iceberg/deltalake table from glue

### PR DESCRIPTION
**Pull request by letian-jiang**
Otherwise `getTable` throws exception

```java
public Table getTable(String dbName, String tableName) {
    org.apache.hadoop.hive.metastore.api.Table table = client.getTable(dbName, tableName);
    StorageDescriptor sd = table.getSd();
    if (sd == null) {
        throw new StarRocksConnectorException("Table is missing storage descriptor");
    }

    ...
}
```

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/connector/hive/glue/converters/CatalogToHiveConverter.java|0|1|00.00%|[155]|

**Review by Youngwb — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/connector/hive/glue/converters/CatalogToHiveConverter.java|0|1|00.00%|[155]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:heart_eyes: **pass** : 11 / 12 (91.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/connector/unified/UnifiedMetadata.java|0|1|00.00%|[65]|
|:large_blue_circle:|com/starrocks/connector/hive/glue/converters/CatalogToHiveConverter.java|11|11|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedMetadata.java [hunk 1]
```diff
@@ -50,15 +50,21 @@ public class UnifiedMetadata implements ConnectorMetadata {
...
 
-    static boolean isDeltaLakeTable(Map<String, String> properties) {
+    public static boolean isDeltaLakeTable(Map<String, String> properties) {
         return DELTA_LAKE_PROVIDER.equalsIgnoreCase(properties.get(SPARK_TABLE_PROVIDER_KEY));
     }
 
+    public static boolean isPaimonTable(Map<String, String> properties) {
+        return PAIMON_STORAGE_HANDLER_VALUE.equalsIgnoreCase(properties.get(PAIMON_STORAGE_HANDLER_KEY));
+    }
+
```
Comment (zombee0) [inline fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedMetadata.java:67]: what about paimon? just defined and no usage. Left LGTM

**Review by zombee0 — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedMetadata.java (hunk 1 above)
Comment (letian-jiang) [reply fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedMetadata.java:67]: reserved for later usage

**Review by stephen-shelby — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Pull request merged**